wp-includes/sqlite/class-wp-sqlite-translator.php

```diff
@@ -1582,15 +1582,15 @@ private function get_primary_keys( $table_name ) {
 	 * @return array
 	 */
 	private function get_keys( $table_name, $only_unique = false ) {
-		$q       = $this->sqlite->query( 'SELECT * FROM pragma_index_list("' . $table_name . '") as l;' );
-		$indices = $q->fetchAll();
+		$query   = $this->sqlite->query( 'SELECT * FROM pragma_index_list("' . $table_name . '") as l;' );
```

Comment:
I'm now thinking all instances of concatenating SQL with PHP variables should be turned into prepared statements